server/dive_server/crud_rpc.py

```diff
@@ -268,6 +271,61 @@ def run_training(
     return newjob.job
 
 
+def process_items(folder: types.GirderModel, user: types.GirderModel):
+    """
+    Discover unprocessed items in a dataset and process them by type in order of creation
+    """
+    unprocessed_items = Folder().childItems(
+        folder,
+        filters={
+            "$and": [
+                # Look for both JSON and CSV items
+                {
+                    "$or": [
+                        {"lowerName": {"$regex": constants.csvRegex}},
+                        {"lowerName": {"$regex": constants.jsonRegex}},
+                    ]
+                },
+                # Don't re-process existing annotation files
+                {f"meta.{constants.DetectionMarker}": {'$not': {"$in": TRUTHY_META_VALUES}}},
+            ]
+        },
+        # Processing order: oldest to newest
+        sort=[("created", pymongo.ASCENDING)],
+    )
+    auxiliary = crud.get_or_create_auxiliary_folder(folder, user)
+    for item in unprocessed_items:
+        file = next(Item().childFiles(item), None)
+        if file is None:
+            raise RestException('Item had no associated files')
+
+        filetype, data, attrs = crud.get_data_by_type(file)
```

Comment:
Errors that occur here (calling this function and the subsequent json.load) can be caused by malformed JSON.  If this is not handled properly by removing the offending JSON file it will stick around and you can't import any other JSON items because it will continually try to process the failing file.